OpenRA.Mods.Common/Lint/CheckSequences.cs

```diff
@@ -18,7 +18,7 @@
 
 namespace OpenRA.Mods.Common.Lint
 {
-	class CheckSequences : ILintPass
+	class CheckSequences : ILintMapPass
```

Comment:
Isn't this supposed to be an `ILintRulesPass`?
Also I see the `map != null` check is still present below, unlike all the other lint checks, why is that?